internal/testing/cluster.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	testBusFlushInterval = 100 * time.Millisecond
 	testPersistInterval  = 2 * time.Second
+	latestHardforkHeight = 50 // foundation hardfork height in testing
```

Comment:
the error shows up for me (consistently) at blocks 45|46 and 52, I was expecting this value to be 52?